slackbot-http/go.mod

```diff
@@ -6,10 +6,12 @@ require (
 	github.com/lusis/go-slackbot v0.0.0-20180109053408-401027ccfef5 // indirect
 	github.com/lusis/slack-test v0.0.0-20190426140909-c40012f20018 // indirect
 	github.com/mozilla-services/foxsec-pipeline-contrib v0.0.0
-	github.com/nlopes/slack v0.5.0
+	github.com/nlopes/slack v0.6.0
 	github.com/sirupsen/logrus v1.4.2
 	go.mozilla.org/mozlog v0.0.0-20170222151521-4bb13139d403 // indirect
-	go.mozilla.org/mozlogrus v1.0.0
+	go.mozilla.org/mozlogrus v2.0.0+incompatible
 )
 
 replace github.com/mozilla-services/foxsec-pipeline-contrib v0.0.0 => ../
+
+go 1.13
```

Comment:
This should be fine. I know the GCP Cloud Function runtime is running go 1.11, but their docs (https://cloud.google.com/go/docs/setup#install_go) make it sound like to doesn't matter (which makes sense, as there haven't been language incompatibility changes afaiu. As well, we deploy using a vendored folder rather than uploading the `go.mod` file, so I believe this should be ok.